apps/shipments/models.py

```diff
@@ -17,9 +18,10 @@
 from enumfields import Enum
 from enumfields import EnumField
 from rest_framework import status
-from rest_framework.exceptions import PermissionDenied
+from rest_framework.exceptions import PermissionDenied, APIException
 from rest_framework.exceptions import ValidationError, Throttled
 from rest_framework.status import HTTP_500_INTERNAL_SERVER_ERROR, HTTP_503_SERVICE_UNAVAILABLE
+from geojson import Feature, LineString, Point as Geo_Point
```

Comment:
I'm surprised lint allowed this - don't use underscores in a class name, even if it's just an alias. But actually I think something like this would be more clear:
```suggestion
import geojson
```
then use `geojson.Feature`, `geojson.LineString`, `geojson.Point`. That way it's less ambiguous in the code, even though it is a little more verbose.